File: LeetCode/80/main.py
class Solution:
    def removeDuplicates(self, arr: List[int]) -> int:
        
        n = len(arr)

        if(n <= 2) :
            return n

        k, c = 1, 0

        for i in range(1, n) :
            if(arr[i] == arr[i-1]) :
                c += 1
            else :
                c = 0
    
            if(c < 2) :
                arr[k] = arr[i]
                k += 1

        return k

File: LeetCode/80/main.cpp
class Solution {
public:
  int removeDuplicates(vector<int> &arr) {
    int n = arr.size();
    if (n <= 2) {
      return n;
    }
    int c = 0, left = 1;

    for (int i = 1; i < n; i++) {
      if (arr[i - 1] == arr[i]) {
        c++;
      } else {
        c = 0;
      }
      if (c < 2) {
        arr[left] = arr[i];
        left++;
      }
    }

    return left;
  }
};